de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ProjectManagementResource.java

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.server.resources;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import de.dlr.sc.virsat.server.configuration.RepositoryConfiguration;
+import de.dlr.sc.virsat.server.controller.RepoManagementController;
+import de.dlr.sc.virsat.server.repository.RepoRegistry;
+
+@Path(ProjectManagementResource.PATH)
+public class ProjectManagementResource {
+
+	public static final String PATH = "/project";
+
+	private RepoManagementController controller;
+
+	public ProjectManagementResource() {
+		controller = new RepoManagementController();
+	}
+
+	@GET
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response getAllProjects() {
+		List<String> projects = new ArrayList<>(RepoRegistry.getInstance().getRepositories().keySet());

```

Comment:
Is it ok to access the RepoRegistry in the Resources or should we move this code into the Controller? E.g. have a function getAllProjects() in the Controller?

Reply:
Yes, makes sense to me. I'll change it.